services/wfm/src/main/java/org/openkilda/wfm/topology/flowhs/fsm/reroute/actions/ValidateFlowAction.java

```diff
@@ -72,6 +72,14 @@ public ValidateFlowAction(PersistenceManager persistenceManager, FlowOperationsD
                 throw new FlowProcessingException(ErrorType.REQUEST_INVALID,
                         format("Flow %s is in progress now", flowId));
             }
+            if (!foundFlow.getSrcSwitch().isActive()) {
```

Comment:
This is not a reliable check. We do not have reliable data about switch state. I would like to fail on the "switch missing" response from FL instead of this check.

What problem we are solving here?